app/locales/sv.json

```diff
@@ -0,0 +1,153 @@
+    "breakTimeReminderApp": "app för rasttidspåminnelse",
+    "chooseLanguage": "För att börja, välj det språk du vill använda med stretchly:",
+    "readTutorial": "Låt stretchy bli din favorit app genom att ta tid att göra den din egen. Lär dig att anpassa den i en snabb handledning.",
+    "tutorial": "handledning",
+    "skipTutorial": "hoppa över handledning, Jag kan alltid hitta den i Om-fönstret"
+  },
+  "tutorial": {
+    "title": "stretchly-handledning",
+    "welcomeTitle": "stretchly-handledning",
+    "welcomeText": "Att ta regelbundna raster och mikroraster gör dig effektivare - det är vetenskap. Men bara du vet ditt flöde. Lär känna stretchly med denna snabba handledning och gör det på ditt sätt.",
+    "defaultsTitle": "",
+    "defaultsText": "stretchly är tänkt att vara diskret, så du kommer inte att se några fönster när du startar den. Vad du får som standard är 20 sekunder mikro rast var 10 minuter och 5 minuters rast efter 2 mikroraster.",
+    "breaksTitle": "",
+    "breaksText": "Rastfönstret innehåller idéer som hjälper dig att slappna av. Du kommer också att se återstående tid. Du kan hoppa över det, men varför skulle du? Raster hjälper till bättre stresskontroll, lindrar dina trötta ögon och ger dig chansen att flytta din kropp. Varför hoppa över chansen att vara mer fokuserad och kreativ?",
+    "notificationsTitle": "",
+    "notificationsText": "Avisering visas 10 sekunder före mikrorast och 30 sekunder före rast, så du kan undvika frestelsen att hoppa över pausen eftersom du <i>verkligen-behöver-bara-30-sekunder-för-att-klara-detta</i>.",
+    "tray1Title": "",
+    "tray1Text": "Vill du se när nästa rast kommer? Håll muspekaren över ikonen i systemfältet.",
+    "tray2Title": "",
+    "tray2Text": "Är ditt flöde annorlunda? Ska du presentera något? Dröj stretchly för en tidsperiod. Behöver du en snabb rastidé? Klicka på systemfältsmenyn och du kan göra allt det. Även återställa raster som du just har börjat.",
+    "settings1Title": "",
+    "settings1Text": "Om du märker att du ständigt hoppa över raster, ändra inställningar och välj tid och raster som passar dig. Eller sätt på så kallat 'striktläge' där du inte får hoppa över raster. Din kropp och ditt sinne kommer att tacka dig. ",
+    "settings2Title": "",
+    "settings2Text": "Vi älskar alla färgscheman och ljud för slutet av pausen, så vi kunde inte bestämma vilken som skulle sättas som standard. Hjälp oss och välj din favorit, så de inte glöms.",
+    "settings3Title": "",
+    "settings3Text": "På sista inställningssidan kan du ändra språk för gränssnittet och några extra inställningar. Du behöver nog inte ändra dem, men hej, varför inte kolla in dem?",
+    "thanksTitle": "Tack!",
+    "thanksText": "Varje månad tar det mycket arbete att påminna dig om att ta en paus. Vi hoppas att stretchly hjälper dig att leda ett mer balanserat liv. Om det är så att du kan stödja mig med "Bli en beskyddare (Patron)"-knappen i systemfältsmenyn."
```

Comment:
One more here ;]  Also, please add yourself to readme: https://github.com/hovancik/stretchly#contributors 

You could also update CHANGELOG. If not sure, I can do it later.